iree/compiler/Dialect/HAL/Conversion/FlowToHAL/ConvertStreamOps.cpp

```diff
@@ -560,11 +573,22 @@ static void allocateTransientBuffers(IREE::Flow::ExStreamFragmentOp streamOp,
         subspanOp.getLoc(), bufferValue.getType(), bufferValue, offsetValue,
         lengthValue);
     auto bufferRange = BufferRange{subspanValue, lengthValue};
-    schedulingState.mapTensorToBufferRange(tensorValue, bufferRange);
+    if (failed(
+            schedulingState.mapTensorToBufferRange(tensorValue, bufferRange))) {
+      llvm::errs() << subspanOp << "\n";
+      return WalkResult::interrupt();
```

Comment:
Is this (calling `llvm::errs()` instead of `emitOpError()`) intended / a standard way to report errors?

Reply:
lolol no. This is a random debugging statement I forgot to delete :see_no_evil:

Reply:
Hah :P